README.md

```diff
@@ -9,20 +9,23 @@ These instructions will get you a copy of the project up and running on your loc
 
 ### Prerequisites
 
-* [CENTOS 6/7](https://wiki.centos.org/)
-* [PHP 7.1](http://php.net/docs.php )
-* [Symfony 3.4](https://symfony.com/doc/3.4/index.html) - PHP framework for web application
-* [PostgreSQL 9.6](https://www.postgresql.org/docs/9.6/static/release-9-6.html) - ORDBMS
+* [Apache 2.4](https://www.apache.org) - Webserver
```

Comment:
Not a big fan of a alphabetized listing, it should be be significance or install order rather.

Reply:
re-ordering